build/build.proj

```diff
@@ -355,19 +355,19 @@
 
     <PropertyGroup>
       <!-- Sort assemblies -->
-      <DesktopInputTestAssemblies>@(TestAssemblyPath->WithMetadataValue("IsDesktop", "true"))</DesktopInputTestAssemblies>
-      <DesktopInputTestAssembliesSpaced>$(DesktopInputTestAssemblies.Replace(';', ' '))</DesktopInputTestAssembliesSpaced>
-      <CoreInputTestAssemblies>@(TestAssemblyPath->WithMetadataValue("IsCore", "true"))</CoreInputTestAssemblies>
-      <CoreInputTestAssembliesSpaced>$(CoreInputTestAssemblies.Replace(';', ' '))</CoreInputTestAssembliesSpaced>
+      <DesktopInputTestAssemblies Condition=" '$(RunTestType)' == 'IsDesktop' ">@(TestAssemblyPath->WithMetadataValue("IsDesktop", "true"))</DesktopInputTestAssemblies>
+      <DesktopInputTestAssembliesSpaced Condition=" '$(RunTestType)' == 'IsDesktop' ">$(DesktopInputTestAssemblies.Replace(';', ' '))</DesktopInputTestAssembliesSpaced>
+      <CoreInputTestAssemblies Condition=" '$(RunTestType)' == 'IsCore' ">@(TestAssemblyPath->WithMetadataValue("IsCore", "true"))</CoreInputTestAssemblies>
+      <CoreInputTestAssembliesSpaced Condition=" '$(RunTestType)' == 'IsCore' ">$(CoreInputTestAssemblies.Replace(';', ' '))</CoreInputTestAssembliesSpaced>
```

Comment:
How is Heng adding new tests for netstandard2.1? Assuming it's a netcoreapp3.0 app, then that will also be `IsCore`, so we're going to have one of the phases run twice as long as the other?  `IsDesktop` tests net472 and `IsCore` tests both netcoreapp2 and netcoreapp3?

Reply:
Desktop takes longer now anyawys, and she's not adding netcoreapp3.0 for every assembly. 

But I'd be curious to see the results of the 2 changes integrated.

Reply:
I have a separate branch testing Heng's 3.0 changes. As indicated in this PR's description, we may learn some better ways of splitting things up after this split runs for some time.